nondex-maven-plugin/src/it/clean-it/pom.xml

```diff
@@ -38,11 +35,6 @@
       <version>4.12</version>
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>@project.groupId@</groupId>
-      <artifactId>nondex-annotations</artifactId>
-      <version>@project.version@</version>
```

Comment:
was there a reason that clean-it needed `nondex-annotations` and `OtherIgnore`, but no longer does? Or was it just always not needed?